server/server.go

```diff
@@ -70,31 +70,12 @@ func NewServer(ctx *Context) (*Server, error) {
 	if ctx == nil {
 		return nil, util.Error("ctx must not be null")
 	}
-	// Determine hostname in case it hasn't been specified in -rpc or -http.
+	// Determine hostname in case it hasn't been specified in -http.
 	host, err := os.Hostname()
```

Comment:
`rpc.Server.Addr` is the resolved address and will have the form `ip-address:port`. Do you think it is important to have the hostname instead of an ip-address?